pkg/client/deploymentconfigs.go

```diff
@@ -23,6 +24,8 @@ type DeploymentConfigInterface interface {
 	Watch(label labels.Selector, field fields.Selector, resourceVersion string) (watch.Interface, error)
 	Generate(name string) (*deployapi.DeploymentConfig, error)
 	Rollback(config *deployapi.DeploymentConfigRollback) (*deployapi.DeploymentConfig, error)
+	GetScale(name string) (*extensions.Scale, error)
```

Comment:
@ironcladlou would you prefer a `DeploymentConfigScalerInterface`?  I don't feel strongly.